src/main.rs

```diff
@@ -873,6 +888,54 @@ fn kaluga_hello_world(
     }
 }
 
+#[cfg(feature = "aztouchmod")]
+fn aztouchmod_hello_world(
+    backlight: gpio::Gpio15<gpio::Unknown>,
+    dc: gpio::Gpio4<gpio::Unknown>,
+    rst: gpio::Gpio22<gpio::Unknown>,
+    spi: spi::SPI2,
+    sclk: gpio::Gpio18<gpio::Unknown>,
+    sdo: gpio::Gpio23<gpio::Unknown>,
+    cs: gpio::Gpio5<gpio::Unknown>,
+) -> Result<()> {
+    info!(
+        "About to initialize the AZTouchMod ILI9341 SPI LED driver",
+    );
+
+    let config = <spi::config::Config as Default>::default()
+        .baudrate(2.MHz().into());
```

Comment:
```suggestion
        .baudrate(26.MHz().into());
```

Oh wait, this is too slow. I reduced the speed to 2Mhz while debugging an issue I had, in the end I had just exchanged the GPIOs for MISO and MOSI..

sending an update to the patch